Modules/LearningSequence/classes/GlobalScreen/class.ilLSViewLayoutProvider.php

```diff
@@ -136,7 +136,7 @@ public function getContentModification(CalledContexts $screen_context_stack) : ?
         // away the header here.
         return $this->globalScreen()->layout()->factory()->content()
             ->withModification(
-                function (Legacy $content) use ($html) : Legacy {
+                function (?Legacy $content) use ($html) : Legacy {
```

Comment:
@chfsx This should return `?Legacy` as well, right?

Reply:
No, just `Legacy` seems to be fine: https://github.com/ILIAS-eLearning/ILIAS/blob/release_8/src/GlobalScreen/Scope/Layout/Factory/ContentModification.php#L65